scripts/write-release-notes.ps1

```diff
@@ -103,7 +103,7 @@ See the release notes [here](https://github.com/microsoft/vstest-docs/blob/maste
 
 ## $v
 
-### Issue Fixed
+### Issues Fixed
```

Comment:
```suggestion
### $(if ($issues.Length -eq 1) { 'Issue' } else { 'Issues' }) Fixed
```

Reply:
I considered it and decided it was overkill, but you prefer, sure.